examples/ipfs-camp-2019/05-Discovery/protocol.go

```diff
@@ -51,7 +51,7 @@ func chatInputLoop(ctx context.Context, h host.Host, donec chan struct{}) {
 	for scanner.Scan() {
 		msg := scanner.Text()
 		for _, peer := range h.Network().Peers() {
-			if _, err := h.Peerstore().SupportsProtocols(peer, string(chatProtocol)); err == nil {
+			if _, err := h.Peerstore().SupportsProtocols(peer, chatProtocol); err == nil {
```

Comment:
Don't update the examples. They have a separate `go.mod` file. We only update them once we release a new libp2p version.

Reply:
fixed this.

Reply:
Am I right to assume you forgot to push?

Reply:
sorry about that. fixed. I had updated the other branch sukunrt/fix-pid-string-conv-ms-merged